corporate/lib/stripe.py

```diff
@@ -2873,8 +2873,17 @@ def approve_sponsorship(self) -> str:
             self.write_to_audit_log(
                 event_type=AuditLogEventType.SPONSORSHIP_APPROVED, event_time=timezone_now()
             )
-        # TODO: Add something (probably email) to let remote realm
-        # organization know that the sponsorship request was approved.
+        send_email(
+            "zerver/emails/sponsorship_approved_community_plan",
+            to_emails=[self.remote_realm.server.contact_email],
+            from_address=FromAddress.SUPPORT,
+            context={
+                "billing_entity": self.billing_entity_display_name,
+                "plans_link": f"{self.billing_session_url}/plans/",
```

Comment:
So because you can't actually log in using these URLs, I'd be tempted to just include a `zulip.com/plans#self-hosted` link rather than the one on their login page -- since the sessions expire fairly quickly, I expect most users will not have an active session when they click this link.

I guess if they use the same browser they used to sign up, which is not at all guaranteed but probably unlikely (in that the login flow involved clicking a button from email in the first place), we will do a fancy redirect to their login flow, but I worry that if they use a different one.

Reply:
I wasn't sure about which to use, but this makes sense.